### PR TITLE
Update application.yml

### PR DESCRIPTION
**Pull request by wwelling**
# Description

Fixes create-drop and initialization of Postgres

## Type of change

Property changes.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -57,8 +61,8 @@ spring:
     defer-datasource-initialization: true
 
     database-platform: org.hibernate.dialect.H2Dialect
-    #database-platform: org.hibernate.dialect.PostgreSQLDialect
-    #database-platform: org.hibernate.dialect.MySQL5InnoDBDialect
+    # database-platform: org.hibernate.dialect.PostgreSQL95Dialect
```
Comment (kaladay) [inline src/main/resources/application.yml:64]: Postgresql 9 itself is being deprecated by hibernate.
- https://hibernate.atlassian.net/browse/HHH-15487

We should add a comment about `PostgreSQL95Dialect` and similar being used for the appropriate 9.x version but the `PostgreSQLDialect` requires a postgresql of at least 10.x.

Comment (wwelling) [reply src/main/resources/application.yml:63]: Added comment. Thanks @kaladay

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -114,7 +117,7 @@ app:
     duration: 3600000
 
   # edu.tamu.weaver.auth.service.UserCredentialsService
-  authority.admins:
+  authority.admins: jcreel@tamu.edu, jmicah@tamu.edu, huff@tamu.edu, wwelling@tamu.edu
```
Comment (jcreel) [inline src/main/resources/application.yml:120]: Let's leave the emails off of the public repo for now

Comment (wwelling) [reply src/main/resources/application.yml:120]: Okay. Removing.

**Review by jcreel — APPROVED** [review]

**Pull request merged**